### DISCUSSION TITLE
a way to install an extension on a managed chromebook

### DISCUSSION THREAD

**Discussion by raahilaalam** [3kh0/ext-remover#1322] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
you can go on an microsoft or windows computer then install chrome,sign in your school email,then you can install an extension.THEN turn on sync.You will have the extension on a enrolled chromebook.

**Comment by ghost** [comment]
sorry, i don't really understand what you mean. If I were to go onto my Microsoft computer with Chrome, sign in with my school email and install an extension with sync enabled, will the extension be on the Chromebook or just the PC?

**Reply by akabutnicer** [reply]
nah it works on iphones

**Reply by ghost** [reply]
it does!!!?????

**Reply by akabutnicer** [reply]
😀👍

**Comment by ghost** [comment]
I HAVE TO TRY IT

**Comment by Charlieis1nsan3** [comment]
it no work

**Reply by akabutnicer** [reply]
it no wok

**Comment by S-PScripts** [comment]
didn't work for me anyways im closing

**Reply by Charlieis1nsan3** [reply]
Because of the policy school forcefully only allow extensions on the allowlist, it is physically impossible to get a extension thats not on the allowlist. (Unless you unenrolled)
